kujira/assetlist.json

```diff
@@ -390,6 +390,31 @@
         }
       ]
     },
+    {
+      "description": "Quark's auto-compounding AQLA derivative",
+      "denom_units": [
+        {
+          "denom": "factory/kujira1d9fmsgvu3yvvppvqech2s3dhrqdm5ukngcf82ghcl70edfpwcrlsu6yr0u/urcpt",
+          "exponent": 0
+        },
+        {
+          "denom": "qcAQLA",
+          "exponent": 6
+        }
+      ],
+      "base": "factory/kujira1d9fmsgvu3yvvppvqech2s3dhrqdm5ukngcf82ghcl70edfpwcrlsu6yr0u/urcpt",
+      "name": "qcAQLA",
+      "display": "qcAQLA",
+      "symbol": "qcAQLA",
```

Comment:
since it's a derivative, please define the `traces`.

Reply:
Hey, can you give an example of a non-IBC transition that would fit the trace?

Previous Quark assets (and indeed, all derivatives on Kujira) don't have this information.

Reply:
I guess it depends on the type of derivative. If it's a burn receipt, or leveraged version, or an index token, then there's no single 1:1 asset to refer to. But in cases of being transferred version or synthetic version of a single asset, then there should be traces. It's hard to say when I don't know what the relationship of qcAQLA to AQLA is.

Reply:
It’s a LSD (a la stATOM). I suppose I’d trace back to the original AQLA?

Do you have an example of a non-IBC trace?

Reply:
Added a trace.